site/src/components/popover.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Popover
+description: The Popover component in the Atlas Design System
+template: standard
+---
+
+# Popover
+
+The popover component allows both simple and more complex content to be placed in the popup that appears after clicking on a button. Popover is based on top of the [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#customizing_the_disclosure_widget) HTML component.
```

Comment:
Suggestion only. remove a few words: ~~both simple and complex~~

Reply:
Updated